solidity/contracts/test/deposit/TestDeposit.sol

```diff
@@ -243,6 +243,9 @@ contract TestDeposit is Deposit {
         return self.withdrawalRequestTime;
     }
 
+    function getCollateralizationPercentage() public view returns (uint256) {
+        return self.getCollateralizationPercentage();
+    }
```

Comment:
This is public now yah?

Reply:
yeah, and It will be here too once we merge master